client/lib/jetpack/use-track-callback.ts

```diff
@@ -6,7 +6,11 @@ import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
 // eslint-disable-next-line @typescript-eslint/no-empty-function
 const noop = () => {};
 
-const useTrackCallback = ( callback: Function = noop, eventName: string, eventProps = {} ) => {
+const useTrackCallback = (
+	callback: ( ...args: any[] ) => void = noop,
```

Comment:
Do we need any or can we prefer the safer alternative `unknown`? 
```suggestion
	callback: ( ...args: unknown[] ) => void = noop,
```